cpp/oneapi/dal/algo/linear_kernel/backend/gpu/compute_kernel_dense_dpc.cpp

```diff
@@ -39,7 +39,7 @@ static compute_result call_daal_kernel(const context_gpu& ctx,
                                        const descriptor_base& desc,
                                        const table& x,
                                        const table& y) {
-    auto& queue = ctx.get_queue();
+    auto queue = ctx.get_queue();
```

Comment:
I'd recommend to use reference here, because `.get_queue()` returns `sycl::queue&`, there is no reason to create a copy.

Reply:
Applied